### PR TITLE
Feat/upgrade atlaskit

### PR DESCRIPTION
**Pull request by Crash--**
Commits & comments should be enough to understand what is going on on this PR!

### CODE REVIEW

**Comment by Crash--** [comment]
@nono I asked you to review this PR so that you would be aware of the change in the Schema.

File: src/lib/collab/schema.js [hunk 1]
```diff
@@ -1,26 +1,49 @@
...
+
+//https://github.com/cozy/cozy-notes/pull/2/commits/202e529cdb1e71996c2cab43057984c9f885b61a
+
+export const nodes = [
   [
     'doc',
     {
       content: '(block)+',
-      marks: 'link'
+      marks: 'link unsupportedMark unsupportedNodeAttribute'
```
Comment (nono) [inline src/lib/collab/schema.js:36]: Do you know what are `unsupportedMark` and `unsupportedNodeAttribute`?

Comment (Crash--) [reply src/lib/collab/schema.js:36]: Nop. The changelog is here https://atlaskit.atlassian.com/packages/editor/editor-core/changelog/%3E=114 I didn't take the time to read it yet

Comment (Crash--) [reply src/lib/collab/schema.js:36]: It seems that is there to preserve nodes & marks even if they are not supported by the fragment

File: src/lib/collab/schema.js [hunk 2]
```diff
@@ -1,26 +1,49 @@
...
     }
   ],
   [
     'paragraph',
     {
+      selectable: false,
```
Comment (ptbrowne) [inline src/lib/collab/schema.js:42]: Why do we need to put selectable: false ?

**Review by ptbrowne — APPROVED** [review]

File: src/lib/collab/schema.js (hunk 2 above)
Comment (Crash--) [reply src/lib/collab/schema.js:42]: I don't know... yet. To be honest, I don't know why we need to dump this internal schema from atlaskit yet.

**Pull request merged**